sample/build.gradle

```diff
@@ -36,6 +36,25 @@ android {
 //            shrinkResources false
 //            proguardFiles getDefaultProguardFile('proguard-android-optim    ize.txt'), 'proguard-rules.pro'
 //        }
+        all {
+
+            def customUserId = System.getenv('STREAM_CHAT_USER_ID')
+            def customUserToken = System.getenv('STREAM_CHAT_USER_TOKEN')
+            def customUserName = System.getenv('STREAM_CHAT_USER_NAME')
+            def customUserImage = System.getenv('STREAM_CHAT_USER_IMAGE')
```

Comment:
we should include API KEY in there as well

Reply:
Done